pypeit/spectrographs/spectrograph.py

```diff
@@ -773,6 +771,7 @@ def slit2order(self, slit_spat_pos):
             return self.orders[iorder], indx[iorder]
 
 
+    # TODO : This code needs serious work.  e.g. eliminate the try/except
```

Comment:
I'm not following why this was changed, and why the documentation describing what this function does is now missing.

Reply:
I think this is another change that slipped in from @profxj 's other branch.  Any comment on this @profxj  ?